wdl_runner/workflows/wdl_pipeline.yaml

```diff
@@ -15,7 +15,7 @@ inputParameters:
   description: Cloud Storage path for output files
 
 docker:
-  imageName: gcr.io/broad-dsde-outreach/wdl_runner:2017_04_09
+  imageName: gcr.io/broad-dsde-outreach/wdl_runner:2017_04_11
```

Comment:
Fairly certain the date was when the image was first created--It wasn't pushed until last week.